Makefile

```diff
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 # Current Operator version
-VERSION ?= 0.0.7
```

Comment:
this and the below changes don't hurt, but I find that they should not be needed. perhaps it;'s better to leave the originally generated file so to make it easier to compare with what new versions of operator sdk generate.

Reply:
I updated it to 0.0.7 myself, so moving back to `latest` was resetting to the default